wave-fe/src/app/game-info/game-info.component.ts

```diff
@@ -14,8 +14,8 @@ export class GameInfoComponent implements OnInit {
   userId$: Observable<any>;
   round$: Observable<any>;
 
-  constructor(private round: RoundService, auth: AuthService) {
-    this.round$ = this.round.round$;
+  constructor(private roundService: RoundService, auth: AuthService) {
+    this.round$ = this.roundService.round$;
```

Comment:
I don't know, maybe better to let consumer who need access to round observable to inject roundService

Reply:
I don't understand your comment :|

Reply:
wrong place

Reply:
never mind, it is correct, I was confusing services